deploy/crds/kieapp.crd.yaml

```
@@ -1913,6 +1913,48 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpmEventEmitters:
+                          description: KafkaJBPMEventEmittersObject kafka configuration
+                            to be used by the KieApp for jBPM Emitter
+                          properties:
+                            acks:
+                              description: The number of acknowledgments that the
+                                emitter requires the leader to have received before
+                                considering a request to be complete, not set by default.
+                              type: integer
+                            bootstrapServers:
+                              description: Comma separated list of host/port pairs
+                                to use for establishing the initial connection to
+                                the Kafka cluster.
+                              type: string
+                            casesTopicName:
+                              description: Name to override the default Case topic
+                                name
```

Improve topic name description, please apply to all topics.
```suggestion
                              description: The topic name for cases event messages. Set up to override the default value jbpm-cases-events. 
```

---

ok